tutorials/Python/neonUtilities/neonUtilitiesPython.md

```diff
@@ -0,0 +1,342 @@
+
+---
+syncID:63993d7274f84e51a1047e1fea9aece1
+title: "Using NEON Utilities in Python"
+description: "Use the neonUtilities R package in Python, via the rpy2 library."
+dateCreated: 2018-12-08
+authors: Claire K. Lunch
+contributors: 
+estimatedTime: 
+packagesLibraries: rpy2
+topics: data-management,rep-sci
+languagesTool: python
+dataProduct: 
+code1: /Python/neonUtilities/neonUtilitiesPython.ipynb
+tutorialSeries: 
+urlTitle: utilities-python
+---
+
+## Install and set up
+
+The instructions below will guide you through using the neonUtilities R package in Python, via the rpy2 package. rpy2 creates an R environment you can interact with from Python. The focus in this tutorial is on the Python implementation, rather than a comprehensive overview of the package itself. For more information about the package, and instructions for running it in R directly, see the readme for the package on the <a href="https://github.com/NEONScience/NEON-utilities/tree/master/neonUtilities" target="_blank">NEON-utilities GitHub repo</a>, and the tutorial on the <a href="http://www.neonscience.org/neonDataStackR" target="_blank">NEON Data Skills page</a>.
+
+Before starting, you will need:
+1. Python 3 installed. It is probably possible to use this workflow in Python 2, but these instructions were developed and tested using 3.6.4.
+2. R installed. You don't need to have ever used it directly. We tested using R 3.4.3, but most other recent versions should also work.
+3. rpy2 installed. Run the line below from the command line. It won't run within Jupyter, the error message is shown for clarity. It also likely will not work on Windows, see next section if you are running Windows.
+
+
+```python
+pip install rpy2
+```
+
+    
+    The following command must be run outside of the IPython shell:
```

Comment:
I can't, that's the error message reply from Python. I could edit it in the md, but it would get overwritten any time I regenerate the md.

Reply:
Since this command shouldn't be run in Notebooks, can you suppress this code chunk from being evaluated (like eval=FALSE in RMD).  Then the error message wouldn't show and that better aligns with what the user will see as well.